ixmp/core.py

```diff
@@ -312,6 +314,25 @@ def add_region_synomym(self, region, mapped_to):
         else:
             _logger_region_exists(_regions, region)
 
+    def check_access(self, user, models, access="view"):
```

Comment:
please use single hyphens `'view'`  for consistency

Reply:
done